src/views/List.vue

```diff
@@ -1,40 +1,42 @@
 <template>
   <div>
     <h1>List</h1>
-    <p>{{list}}</p>
-    <p>{{list.listItems}}</p>
-    <v-data-table
-      :headers="headers"
-      :items="list.listItems"
-      :items-per-page="5"
-      class="elevation-1"
-    ></v-data-table>
+    <p>{{theList}}</p>
+    <p>{{theList.listItems}}</p>
+    <ol style="list-style-type:none;">
+      <li v-for="(item,index) in theList.listItems" :key="`${item.text}${index}`">
+        <list-item :listItem="item"/>
+      </li>
+    </ol>
   </div>
 </template>
 <script>
+import { mapGetters } from 'vuex';
+import ListItem from '@/components/ListItem.vue';
+
 export default {
   name: 'List',
   props: ['title'],
+  components: {
+    ListItem,
+  },
   data() {
     return {
-      headers: [
-        {
-          text: 'item',
-          align: 'start',
-          sortable: true,
-          value: 'text',
-        },
-        {
-          text: 'state',
-          value: 'state',
-        },
-      ],
     };
   },
   computed: {
-    list() {
-      return this.$store.getters.list(this.title);
+    ...mapGetters(['list']),
+    theList() {
```

Comment:
Don't use `the<exisitng_variable>` naming. For readers of this code, It appears to be a duplicated variable, for no apparent reason. One solution is to rename your getter to something else, like `storedList` and/or rename this to something else. 

My preferred solution is this:
Your getter should probably be renamed so it reflects the fact that it's a getter. something like `getList` instead of `list`. This will make your code easier to read, and easier for everyone to find out where this method  is defined.

Here is a really good reference for Vuex conventions: https://docs.vuestorefront.io/guide/vuex/vuex-conventions.html